Content.Server/Ninja/Systems/NinjaGlovesSystem.cs

```diff
@@ -0,0 +1,304 @@
+using Content.Server.Actions;
+using Content.Server.Chat.Systems;
+using Content.Server.Communications;
+using Content.Server.Doors.Systems;
+using Content.Server.DoAfter;
+using Content.Server.Electrocution;
+using Content.Server.GameTicking;
+using Content.Server.GameTicking.Rules;
+using Content.Server.Popups;
+using Content.Server.Power.Components;
+using Content.Shared.Actions;
+using Content.Shared.Administration.Logs;
+using Content.Shared.Damage.Components;
+using Content.Shared.Database;
+using Content.Shared.DoAfter;
+using Content.Shared.Doors.Components;
+using Content.Shared.Emag.Systems;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Interaction;
+using Content.Shared.Ninja.Components;
+using Content.Shared.Ninja.Systems;
+using Content.Shared.Popups;
+using Content.Shared.Research.Components;
+using Content.Shared.Tag;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.Ninja.Systems;
+
+public sealed class NinjaGlovesSystem : SharedNinjaGlovesSystem
+{
```

Comment:
You should really have all of this code in shared. All of the dependencies have a shared version.